invenio_app_rdm/requests_ui/views/requests.py

```diff
@@ -305,3 +311,16 @@ def community_dashboard_request_view(request, community, community_ui, **kwargs)
             user_avatar=avatar,
             include_deleted=False,
         )
+
+    elif is_subcommunity_invitation_request:
```

Comment:
can be moved as a part of if statement above in line 302, the return is exactly the same, so I think there is no need to copy?